From 282fbe32d3601fcfc326f5aa4d7e327224555b18 Mon Sep 17 00:00:00 2001
From: Dmitry Shachnev <mitya57@gmail.com>
Date: Fri, 5 Jun 2015 20:58:19 +0300
Subject: [PATCH 2/4] conftest.py: Use basic theme instead of deprecated
 default

This will avoid failure because of a warning with Sphinx 1.3.
Fixes #3.
---
 tests/conftest.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/conftest.py b/tests/conftest.py
index 66a62106ba..f4b4dbc2d9 100644
--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -46,7 +46,7 @@
 exclude_patterns = []
 
 pygments_style = 'sphinx'
-html_theme = 'default'
+html_theme = 'basic'
 """
 
 
-- 
2.11.0

